src/lib.rs

```diff
@@ -1,28 +1,75 @@
 use ansi_term::Color::{Blue, Green, Red, Yellow};
-use crates_io_api::{SyncClient, Version};
-use thiserror;
+use reqwest::blocking::Client;
+use reqwest::header;
+use serde_derive::Deserialize;
 
 #[derive(Debug, thiserror::Error)]
-pub enum Error {
+pub enum ErrorKind {
     #[error("Unable to find crate")]
     CrateDoesNotExist,
     #[error("Versions Do not exist on crates.io")]
```

Comment:
everywhere where it says `crates.io`, we want might want to make that a constant. I am hoping one day there will be an easy way to mirror the registry and this could end up configurable

Reply:
Used "registry" instead 🙃